beta/experience-platform-extension/experience-platform-setup.md

```diff
@@ -14,13 +14,27 @@ The Adobe Experience Platform Edge mobile extension is currently in BETA. Use of
 
 To start collecting data in [Adobe Experience Platform](https://experience.adobe.com/platform), an XDM schema and a Dataset need to be created. Follow these steps to get started:
 
-1. [Create a schema](https://docs.adobe.com/content/help/en/experience-platform/xdm/tutorials/create-schema-ui.html)
-2. Assign the XDM ExperienceEvent class to the schema.
-3. Add the following mixins: 
-   * ExperienceEvent Application Details
-   * ExperienceEvent Environment Details
-   * ExperienceEvent Commerce Details
-4. [Create a dataset](https://docs.adobe.com/content/help/en/experience-platform/catalog/datasets/user-guide.html#create) where the data should be sent by using the schema defined earlier.
+1. In the browser, navigate to [Adobe Experience Platform](https://experience.adobe.com/platform) and login with your credentials.
+
+2. Create an [XDM Schema](https://docs.adobe.com/content/help/en/experience-platform/xdm/tutorials/create-schema-ui.html) as follows:
+
+   - From the left panel, select Schemas
+   - Click `Create schema`
+   - Select `XDM Experience event`
+
+   - Add the following existing mixins: 
+
+     - ExperienceEvent Application Details
```

Comment:
This isn't required. It doesn't cause any harm, but it's not used in the AEPSDK sample apps or the Edge Commerce demo app.